Client/src/api/api.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from './constants';
+
+function getLocalData() {
+  const [apiData, setApiData] = useState('');
+  const [currentLocation, setcurrentLocation] = useState();
+  const [token, setToken] = useState();
+  const [weatherdata, setweatherData] = useState();
+
+  const AUTH_TOKEN = token;
```

Comment:
why do you need to recreate this constant on each render? You have `token` value already, aren't you ?